src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -19,7 +19,7 @@
 typedef int (*MPIDI_NM_mpi_init_t) (int rank, int size, int appnum, int *tag_bits,
                                     MPIR_Comm * init_comm, int *n_vcis_provided);
 typedef int (*MPIDI_NM_mpi_finalize_t) (void);
-typedef int (*MPIDI_NM_get_vci_attr_t) (int vci);
+typedef MPIDI_vci_resource_t(*MPIDI_NM_vci_get_resource_info_t) (int vci);
```

Comment:
The previous name `get_vci_attr` is better since it is returning an integer with bits representing various attributes or capabilities.